numpy/core/getlimits.py

```diff
@@ -354,6 +355,15 @@ class finfo:
 
     Notes
     -----
+    As in the IEEE-745 standard, NumPy floating point types use subnormal
+    numbers to fill the gap between 0 and `tiny`. This is achieved by 
+    using bits of the mantissa to allow the representation of smaller 
+    exponents than `minexp`. For example, for 64-bit binary floats in the
+    IEEE-754 standard, the smallest positive value is actually 2**-1074.
+    However, since the number of bits available for the mantissa is 
+    reduced the precision of subnormal floating numbers is also often 
+    reduced significantly.
+    
```

Comment:
I'd recommend moving this to after the next paragraph "For developers of NumPy..." or, better yet, replacing or significantly reducing the text and providing an external reference.

Reply:
Yes, I see your point. I have no objections in moving the paragraph below the next paragraph, of course. I will also think about a shorter version that points to the wiki page.

Reply:
Just an added bit of info you may find helpful: the [numpydoc standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) describes how to add references to external docs etc.